zulipterminal/ui_tools/boxes.py

```diff
@@ -201,17 +201,26 @@ def private_box_view(
             ]
             recipient_info = ", ".join(
                 [
-                    f"{self.model.user_dict[email]['full_name']} <{email}>"
+                    f"""{self.model.user_dict[email]['full_name']} <{email}>"""
+                    for email in self.recipient_emails
+                ]
```

Comment:
There's no change here; it can be removed from the commit.

Reply:
Oops -- will do.